lib/ansible/plugins/callback/profile_tasks.py

```diff
@@ -134,7 +135,7 @@ def playbook_on_stats(self, stats):
         # Print the timings
         for uuid, result in results:
             msg = ''
-            msg="{0:-<70}{1:->9}".format('{0} '.format(result['name']),' {0:.02f}s'.format(result['time']))
+            msg="{0:-<70}{1:->9}".format('{0} '.format(to_bytes(result['name'])),' {0:.02f}s'.format(result['time']))
```

Comment:
We probably don't need explicit conversion here.  Instead, try this:
``` python
msg=u"{0:-<70}{1:->9}".format(u'{0} '.format(result['name']),u' {0:.02f}s'.format(result['time']))
```

* Setting the format strings to be unicode literals instead of byte string literals should solve the initial problem of attempting to convert a unicode string (n the result['name'] field into a byte string).
* We don't need to convert result['name'] anymore because its type now agrees with the type of the format string.